### DISCUSSION TITLE
2005/05/08/%E5%88%9A%E5%8F%91%E7%8E%B0%E7%9A%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#24] [Announcements]
# 2005/05/08/%E5%88%9A%E5%8F%91%E7%8E%B0%E7%9A%84

想用 html 语句直接编写 blog 吗？

https://blog.qizhen.xyz/2005/05/08/%E5%88%9A%E5%8F%91%E7%8E%B0%E7%9A%84

**Comment by ruanqizhen** [comment]
sabrina
2005年05月10日 09:32 编辑
发现了N多有意思的东西，不错不错:)

回复
俊
2005年05月10日 00:33 编辑
男孩子用粉红色也不少啊

回复
sina
2005年05月9日 23:22 编辑
呀呀，really？？？不过偶插件都装好咯。。。呵呵

回复
建建
2005年05月9日 15:41 编辑
可以吗，看看

回复
Ya
2005年05月9日 13:25 编辑
你來yaya家灌水唷 ^^

回复
小＆U
2005年05月9日 12:21 编辑
这是做广告吗？^_*男孩子也用粉红色，很少有哦~~~`

回复
~®§§][][bRiTnEy][][§§®~
2005年05月9日 03:28 编辑
HI .. u commented on my page :)i dont understand anything here but it looks cool :):):)

回复
